Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -8953,7 +8953,7 @@ void WebPageProxy::showContactPicker(IPC::Connection& connection, const Contacts
 void WebPageProxy::showDigitalCredentialsPicker(IPC::Connection& connection, const WebCore::DigitalCredentialsRequestData& requestData, WTF::CompletionHandler<void(Expected<WebCore::DigitalCredentialsResponseData, WebCore::ExceptionData>&&)>&& completionHandler)
 {
     MESSAGE_CHECK_COMPLETION_BASE(
-        protectedPreferences()->digitalCredentialsEnabled(),
+        protectedPreferences()->digitalCredentialsEnabled() && protectedPreferences()->webAuthenticationASEnabled(),
```

Comment:
I think you need an `ENABLE(WEB_AUTHN)` since `digitalCredentialsEnabled()` won't *exist* unless that is true.